config.json

```diff
@@ -261,6 +261,14 @@
         "concepts": ["classes"],
         "prerequisites": ["objects", "functions", "conditionals-ternary"],
         "status": "beta"
+      },
+      {
+        "slug": "regular-chatbot",
+        "name": "Regular Chatbot",
+        "uuid": "dc118a0d-be89-4ea5-a814-e471897d921a",
+        "concepts": ["regular-expressions"],
+        "prerequisites": ["arrays", "classes"],
```

Comment:
in the design.md you also mentioned `objects` so probably good to add that here as well, also `template-strings` should be added as you use them in the exercise solution

Reply:
That's good to know. For future exercises I'll try to evaluate all of the possible prerequisites. Thanks 👍️